packages/main/src/Menu.ts

```diff
@@ -492,15 +406,15 @@ class Menu extends UI5Element {
 		return fragment;
 	}
 
-	_openItemSubMenu(item: MenuItem, opener: HTMLElement) {
+	_openItemSubMenu(item: MenuItem) {
 		const mainMenu = this._findMainMenu(item);
 		mainMenu?.fireEvent<MenuBeforeOpenEventDetail>("before-open", {
 			item,
 		}, false, false);
-		item._subMenu!.showAt(opener);
+		item._subMenu!.showAt(item);
```

Comment:
replace with open/opener

Reply:
This one is the `ui5-menu` `showAt` method so we need to use it.

Reply:
We should still remove it. We should make the menu work with the declarative API. We can also remove the async, as with the new update of the code, the renderFinished wont be needed.